### PR TITLE
Add 4xx & 5xx error pages

### PR DESCRIPTION
**Pull request by masihtak**
**Created error pages for the following HTTP status codes:**
- 403 Forbidden
- 404 Not Found
- 500 Internal Server Error

Fix: #346

### CODE REVIEW

**Comment by libremente** [comment]
Hi @masihtak and thanks for the PR!
I like the error pages style, however I don't understand why there are mods in other files (like footer). Could you please further elaborate your decisions regarding those files? Thanks!

**Comment by masihtak** [comment]
Hi @libremente! Glad to see you liked my project. there's no mods for other files it's because my previews PR https://github.com/italia/developers.italia.it/pull/433

**Comment by libremente** [comment]
> Hi @libremente! Glad to see you liked my project. there's no mods for other files it's because my previous PR #433

I undertand, however I believe that you branched from your master which already had the changes you are proposing in #433 and not the clean master from upstream. 
As such, I'd propose to focus on #433 for now, then once that will be merged we will focus on this one with a proper rebase. Thanks! @masihtak

**Comment by libremente** [comment]
Hi @masihtak, since #433 has been merged, I think we are good to go with this as well

File: 500.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+permalink: /500.html
+---
+
+<div class="container h-100">
+  <div class="row h-100 justify-content-center align-items-center">
+    <div class="col-md-12 my-5">
+      <div class="error text-center">
+        <!-- Error title -->
+        <h2 class="error-code">500</h2>
+        <p class="lead font-weight-bold">Internal Server Error</p>
```
Comment (sebbalex) [inline 500.html]: I suggest to add Home link here too.

**Comment by libremente** [comment]
If @masihtak you could please review the PR as suggested by @sebbalex we can easily merge this soon. Thanks a lot!

File: 500.html (hunk 1 above)
Comment (masihtak) [reply 500.html]: 👍

**Comment by masihtak** [comment]
should I also localized it as well?

**Comment by libremente** [comment]
> Should I also localized it as well?

@masihtak  Don't worry about the localization, let's use plain English. Let me know if you need help in order to rebase this branch. After that we can proceed with the merge!

**Comment by masihtak** [comment]
@libremente  I can localize the files but I'm a bit confused, for the error pages we don't have something special to translate, only the error status code and the 'Home' button. Even In the [IT Wikipedia of HTTP status code](https://it.wikipedia.org/wiki/Codici_di_stato_HTTP), you can see that status codes are International English. Here the localization will cause unnecessarily duplication

**Comment by libremente** [comment]
Hi @masihtak, please see this comment of @sebbalex who explains the matter. 
> From an UX point of view those pages needs to be localised with current language set. I suggest to move these pages under `_pages/$lang` folder and add correct jekyll front matter.
> Furthermore error message and `home` links can be localised as well.

Let me know if you need further hints, thanks a lot!

**Comment by masihtak** [comment]
@sebbalex Sorry for delay, I was sick.
Glad to see you finally come up with this to move them in the root. It seems to be good!

**Pull request merged**